docs/deployment/linux/debian.txt

```diff
@@ -169,6 +178,21 @@ In both instances, replace ``COMMAND`` with ``start``, ``stop``, ``restart``,
 Configuration
 =============
 
+Configuration Files
+-------------------
+
+The main CrateDB configuration files live in the ``/etc/crate`` directory.
+
+.. TIP::
```

Comment:
Not sure really, whether this should be a TIP. Changing an installation configuration provided by a package can be dangerous and lead to all sorts of problems. I would at least mention, that changing the configuration directory may cause unexpected behaviour.
If you want to tinker around, the simpler tarball installation should be used.

Reply:
If you want to see what files have been installed and where they are located, you could use ` dpkg-query -L crate`.